server/data/data-api/src/main/java/org/apache/james/mailrepository/api/MailRepositoryStore.java

```diff
@@ -19,7 +19,6 @@
 

```

Comment:
Why can't the `MailRepositoryStore` allow me to directly list all MailRepositoryPaths available?

Reply:
Because there's no endpoint/usecase for that.

Reply:
There is a use case:

```
GET /mailRepositories
```

As far as I know...